### ISSUE TITLE
Function drain_OD in ent_tank.py has error when running example

### ISSUE THREAD

**Issue by anthonyverghese** [AguaClara/aide_design#47]
When I try creating an example for this function, I get a Traceback error. However, when I do shift+enter on the function as a whole, I get a check mark from hydrogen.

**Comment by monroews** [comment]
Isn't that the expected result? Line by line execution of functions doesn't work in hydrogen.

**Comment by anthonyverghese** [comment]
Yes, probably. Also, I forgot that we don't need to test the files that don't have dict in them
